docs/release-notes/master.adoc

```diff
@@ -13,10 +13,47 @@ include::topics/templates/document-attributes.adoc[]
 //Inclusive language statement
 include::topics/making-open-source-more-inclusive.adoc[]
 
-include::topics/mta-rn-new-features.adoc[leveloffset=+1]
+== Introduction
+{ProductName} {DocInfoProductNumber} accelerates large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
+
+These release notes cover all z-stream releases of {ProductShortName} 6.0 with the most recent release listed first.
+
+== {ProductShortName} 6.0.1
+
+include::topics/mta-rn-new-features-1.adoc[leveloffset=+2]
+
+[id="rn-upgrade-procedures"]
+=== Upgrading {ProductShortName}
```

Comment:
I think "Upgrading MTA" should be under "Support for non-administrators to run MTA"
as the content of  "Upgrading MTA" is not how to upgrade but rather how to use the above improvement when upgrading.